frontend/src/reducers/rooms.js

```diff
@@ -13,7 +13,7 @@ const reducer = (state = initialState, action) => {
         case actionTypes.ADD_ROOM:
             return {
                 ...state,
-                rooms: [...state.rooms, action.room].sort((a, b) => a.name.localeCompare(b.name)),
+                rooms: [...state.rooms, action.room],
```

Comment:
Are you sure added room will be add to the end of array ?

Reply:
fixed